.drone/drone.yml

```diff
@@ -139,6 +139,40 @@ steps:
       include:
       - refs/heads/master
       - refs/tags/v*
+- depends_on:
```

Comment:
It appears to me that drone.yml is generated from drone.jsonnet via the Makefile.  Should these changes be in drone.jsonnet?

Reply:
really ?

Reply:
Yes, definitely! The `drone.yml` should never be edited by hand

Reply:
doh !